module_2/assignment1_3.js

```diff
@@ -0,0 +1,34 @@
+let arr = ['John', 'Ram', 'ari', 'nal'];
+const ARR = [3.147, 'Kiya', 23];
```

Comment:
Camel casing is required for defining an array, please change it.
All capital letter is use when some constant value is defined like PI / any constant message.

Reply:
done